src/app/providers/network-state/network-state.ts

```diff
@@ -38,9 +38,11 @@ export class NetworkStateProvider {
     this.networkStatus$.next(status);
   }
 
+  // eslint-disable-next-line rxjs/finnish
```

Comment:
Is there no potential fix, instead of just disabling linting for that line?

Reply:
I tried looking into enabling/disabling this based on what type of construct it was like how it is in the current mes project but I couldn't find anything